Library/ViewModels/CreatorByLineViewViewModel.swift

```diff
@@ -0,0 +1,44 @@
+import KsApi
+import Prelude
+import ReactiveSwift
+
+public protocol CreatorByLineViewViewModelInputs {
+  func configureWith(project: Project)
+}
+
+public protocol CreatorByLineViewViewModelOutputs {
+  /// Emits an image url to be loaded into the creator's image view.
+  var creatorImageUrl: Signal<URL?, Never> { get }
+
+  /// Emits text to be put into the creator label.
+  var creatorLabelText: Signal<String, Never> { get }
+}
+
+public protocol CreatorByLineViewViewModelType {
+  var inputs: CreatorByLineViewViewModelInputs { get }
+  var outputs: CreatorByLineViewViewModelOutputs { get }
+}
+
+public final class CreatorByLineViewViewModel: CreatorByLineViewViewModelType,
```

Comment:
I know this has happened before in the project but it's not necessary to use `ViewViewModel` for views, this can just be `CreatorBylineViewModel`. I think we have an tech debt task somewhere to rename the other ones to not be `ViewView`.